platform/bitcoin/api_test.go

```diff
@@ -114,8 +124,18 @@ func TestInferDirection(t *testing.T) {
 	set.Add("bc1q2sykr9c342mjpm9mwnps8ksk6e35lz75rpdlfe")
 	set.Add("bc1qs86ucvr3unce2grvfp77433npy66nzha9w0e3c")
 
-	inputs1 := []string{"bc1q2sykr9c342mjpm9mwnps8ksk6e35lz75rpdlfe"}
-	outputs1 := []string{"bc1q6wf7tj62f0uwr6almah3666th2ejefdg72ek6t"}
+	inputs1 := []blockatlas.TxOutput{
+		blockatlas.TxOutput{
```

Comment:
We don't need to define the type again.

```suggestion
		{
```